elements/lisk-p2p/test/integration/maximum_payload.ts

```diff
@@ -34,20 +34,20 @@ describe('Maximum payload', () => {
 		for (let i = 0; i < 6000; i++) {
 			dataLargerThanMaxPayload.push(`message${i}`);
 		}
-		p2pNodeList = [...new Array(NETWORK_PEER_COUNT).keys()].map(index => {
+		p2pNodeList = [...new Array(NETWORK_PEER_COUNT / 2).keys()].map(index => {
```

Comment:
Also it's not clear why only half. It seems more arbitrary and adds an element of randomness depending on which nodes start first.

Reply:
So the reason for such a way of choosing seed peers is, many times the first node tries to make a connection with other nodes that already have an incoming connection with it. It was working before without the hotfix because it was still counting the outbound connection to others even though it was disconnected from there end, but now it throws duplicate connection error and it marks them as disconnected, hence giving the right count of outbound/inbound